src/Commands/Configure.ts

```diff
@@ -44,6 +51,20 @@ export class Configure implements CommandModule {
             })
           )
         });
+
+        if (args.nogui) {
+          container.register(
+            Platform,
+            { useClass: ReadlinePlatform },
+            { lifecycle: Lifecycle.Singleton }
+          );
+        } else {
+          container.register(
+            Platform,
+            { useClass: KoaPlatform },
+            { lifecycle: Lifecycle.Singleton }
+          );
+        }
```

Comment:
```suggestion
        container.register(
            Platform,
            { useClass: args.nogui ? ReadlinePlatform : KoaPlatform },
            { lifecycle: Lifecycle.Singleton }
          );
```

Reply:
Well, I tried to do like this, but got an error
`No overload matches this call.
  Overload 1 of 5, '(token: InjectionToken<unknown>, provider: TokenProvider<unknown>, options?: RegistrationOptions): DependencyContainer', gave the following error.
    Argument of type '{ useClass: typeof ReadlinePlatform | typeof KoaPlatform; }' is not assignable to parameter of type 'TokenProvider<unknown>'.
      Object literal may only specify known properties, and 'useClass' does not exist in type 'TokenProvider<unknown>'.
  Overload 2 of 5, '(token: InjectionToken<ReadlinePlatform>, provider: ClassProvider<ReadlinePlatform>, options?: RegistrationOptions): DependencyContainer', gave the following error.
    Type 'typeof ReadlinePlatform | typeof KoaPlatform' is not assignable to type 'constructor<ReadlinePlatform> | DelayedConstructor<ReadlinePlatform>'.
      Type 'typeof KoaPlatform' is not assignable to type 'constructor<ReadlinePlatform> | DelayedConstructor<ReadlinePlatform>'.
        Types of construct signatures are incompatible.
          Type 'new (routerFactory: KoaRouterFactory) => KoaPlatform' is not assignable to type 'new (...args: any[]) => ReadlinePlatform'.`

Reply:
You just need to use a union type

```ts
register<ReadlinePlatform | KoaPlatform>()
```